frontend/src/views/TE/Task.vue

```diff
@@ -135,6 +137,14 @@ export default class Task extends Mixins<TaskMixin, FormatTime>(
     this.isCoordinator = claims.coordinator;
   }
 
+  onCancelClick() {
+    this.cancelAllotment(this.task, "", {}, this.cancelData());
```

Comment:
This call looks ugly. Isn't it possible to have this method `cancelAllotment` without any parameters and be directly hooked to button's `@click`? As all the necessary info for cancellation is available in the context. In other words, we are not calling this method with other set of parameters, and are not going to.

Reply:
Currently the cancel callback is shared between tasks view and task view. Tasks view uses the assignee callback which gets called with those parameters. Should the cancel callback be unique for the separate views?

Reply:
I think in this case we can have them separate